modules/core/src/main/java/org/apache/ignite/internal/util/IgniteUtils.java

```diff
@@ -1449,6 +1451,39 @@ public static double offheapSize(Iterable<ClusterNode> nodes, int precision) {
         return roundedHeapSize(totalOffheap, precision);
     }
 
+    /**
+     * Gets total allocated persistence size in GB rounded to specified precision.
+     *
+     * @param nodes Nodes.
+     * @param precision Precision.
+     * @return Total allocated PDS size in GB.
+     */
+    public static double allocatedPDSSize(Iterable<ClusterNode> nodes, int precision) {
+        double totalAllocatedPDS = .0;
+
+        for (ClusterNode n : nodesPerJvm(nodes)) {
```

Comment:
Shouldn't we summarize the allocated pds size for each node in one Jvm? Since they will use separate persistent directories.

Reply:
Thank you for this. It's my rude mistake. It's fixed to summarize the allocated PDS size of all non-client nodes.